EventsExpress/EventsExpress.csproj

```diff
@@ -25,7 +25,6 @@
     <PackageReference Include="MediatR" Version="7.0.0" />
     <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="7.0.0" />
     <PackageReference Include="MicroElements.Swashbuckle.FluentValidation" Version="3.1.0" />
-    <PackageReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Why was this removed?

Reply:
This is an obsolete package reference: [Migrate from ASP.NET Core 2.2](https://docs.microsoft.com/en-us/aspnet/core/migration/22-to-30?view=aspnetcore-3.1&tabs=visual-studio#update-the-project-file)